app/views/users/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<% @users.each do |user| %>
+    <%= link_to user do %>
+        <%= render "shared/user_card", user: user %>
+    <% end %>
+<% end %>
```

Comment:
[OPTIONAL]

- It is not a good practice to write logic or loop in views. I have shown you below what is the better practice instead of using `@objects.each`.

```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```